traffic_ops/app/db/migrations/2021011500000000_update_interfaces_multiple_routers.sql

```diff
@@ -0,0 +1,53 @@
+/*
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+-- +goose Up
+-- SQL in section 'Up' is executed when this migration is applied
+ALTER TABLE interface ADD COLUMN router_host_name text DEFAULT '';
+ALTER TABLE interface ADD COLUMN router_port text DEFAULT '';
```

Comment:
I know that this is currently a text field, but isn't it actually a literal port? If so, seems like it should be an integer

Reply:
Checked with Jeff Howell, and he suggested keeping it as a string. I changed the name to be `router_port_name` though.